### DISCUSSION TITLE
2023-10-18 Weekly Meeting 129 Notes/Agenda

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#422] [Weekly Meetings]
Weekly Meeting 129 is happening on Wednesday, October 18, 2023 from 9:00 AM PDT (16:00 UTC) - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat.

Who we are and why we're here:
The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal.

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in the 🔈 omi-weekly-meeting chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

Agenda:

### 1. Roundtable
fun, interesting, random or relevant things people have found this week. Please share in the comments.

### 3. Follow on from last week's meeting
Last week https://github.com/omigroup/omigroup/discussions/421

Demo day 30 October 2023.

Research Show and Tell?, boring? 
How about brainstorm about an Open Metaverse Conference instead?
coinciding with metaverse museum crowdfunder launch. Date changed to 30 October. 
We need a flyer and a virtual venue 

Nominations for OMI chairs now open, closing after this meeting.
Nominate yourself or someone else. Elections on 1 November 2023.
More info on roles of Chairs https://hackmd.io/@mrmetaverse/omi-tasks-audit;
also: https://github.com/omigroup/omigroup/discussions/157
and https://github.com/omigroup/omigroup/discussions/165

Weekly game night progress? Still to do - look at what is out there already and join up with that. Make an itenirary. e.g. look at https://devpost.com/hackathons

### 4. Other

**Comment by indiebio** [comment]
1. Roundtable

**Reply by indiebio** [reply]
jin: https://twitter.com/m3org/status/1714105623263039889 some stuff we been up to in m3

**Reply by indiebio** [reply]
jin: 
I fixed the omi-archive action runner I think
adding notes from past meetings
Will have enough data for someone to make AI bot with, perhaps using magickml
Same for M3 and MSF
Discord bot for each would be most excellent

- What each group is doing
- Events coming up
- How to contribute

uploaded summaries of 10 more OMI meetups, there are spelling issues throughoutt them - it'd be nice if someone went through and fixed. Will upload the markdown files to omi-archive on github. For now, they're here as well: https://hackmd.io/@XR/omigroup/https%3A%2F%2Fhackmd.io%2F%40XR%2Fomi_10-12-23

all the notes are here: https://github.com/omigroup/omi-archive
https://omigroup.github.io/omi-archive/omigroup/call_notes/index.html github pages

**Reply by indiebio** [reply]
Knev and myself are sharing ideas and looking for academic funding. Early days:

We are exploring building a pervasive game about grassroots urban interventions.

My background and interest is urban water management, and resource management more generally.

I became interested in a game after the extreme drought in Cape Town, South Africa, where I'm from, when people were not able to coherently coordinate grassroots initiatives that still made sense from a large scale perspective. There was also a rampant spread of misinformation and conspiracy theories, which baffled me.  The drought happened around 2014 – 2018, which coincided with Trump’s election and hence there was a lot of writing about why people were behaving this way.

One of the articles that really stuck with me was about curiosity:
“Scientifically literate people, remember, were more likely to be polarised in their answers to politically charged scientific
questions. But scientifically curious people were not. _**Curiosity brought people together in a way that mere facts did not.**_ “ – Tim Harford, 2017. [[https://timharford.com/2017/03/the-problem-with-facts/](https://timharford.com/2017/03/the-problem-with-facts/](https://timharford.com/2017/03/the-problem-with-facts/%5D(https://timharford.com/2017/03/the-problem-with-facts/)
"https://timharford.com/2017/03/the-problem-with-facts/")

And so I thought the best way to cultivate curiosity, to gently bring people back to critical thinking and appropriate action, is a game.

We are exploring funding opportunities and need to build a consortium.

This links to the metaverse in the integration of data through translators that Knev.phd is working on. And the general ideology of the metaverse, obviously.

**Comment by mrmetaverse** [comment]
# Election 
- nominations would have ended today 
  - do we need another week for nominations? (yes)
  - create an event on discord set to end next week 
  - create a space in omi-votes to nominate. It will be a thread. 
- elections on November 1, 2023 
- next steps post-election: New chairs to set a meeting cadence, merge W3C and Discord lists and begin active engaging and updates, contact OMF to discuss collab or merge, contact the Open Collective staff to make the official switch, update all contact info on all public sites. 
- action items: 
  - create an event for the deadline for nominations (on discord) 
  - create an event for the election day itself.

**Reply by mrmetaverse** [reply]
nominations happening now on discord! https://discord.gg/MwQsTE2c?event=1164246372384518235

**Comment by mrmetaverse** [comment]
# Magick 

We also discussed  [Magick ML](https://magickml.com) and potentially making agents for discord for questions about our recent efforts, upcoming events, working groups, and more! 

Point of order: @mrmetaverse is cofounder of Magick ML and serves as chair at OMI. 

Fun fact: OMI_Personality is a pretty sweet extension we should play with ;)
